scripts/eurostat/health_determinants/physical_activity/process.py

```diff
@@ -0,0 +1,860 @@
+            df = pd.read_csv(file_path, sep='\t', header=0)
+            # Taking the File name out of the complete file address
+            # Used -1 to pickup the last part which is file name
+            # Read till -4 inorder to remove the .tsv extension
+            file_name = file_path.split("/")[-1][:-4]
+            function_dict = {
+                "hlth_ehis_pe9e":
+                    healthenhancing_by_sex_education,
+                "hlth_ehis_pe9i":
+                    healthenhancing_by_sex_income,
+                "hlth_ehis_pe9u":
+                    healthenhancing_by_sex_urbanisation,
+                "hlth_ehis_pe1e":
+                    workrelated_by_sex_education,
+                "hlth_ehis_pe1i":
+                    workrelated_by_sex_income,
+                "hlth_ehis_pe1u":
+                    workrelated_by_sex_urbanisation,
+                "hlth_ehis_pe3e":
+                    nonworkrelated_by_sex_education,
+                "hlth_ehis_pe3i":
+                    nonworkrelated_by_sex_income,
+                "hlth_ehis_pe3u":
+                    nonworkrelated_by_sex_urbanisation,
+                "hlth_ehis_pe2e":
+                    healthenhancing_nonworkrelated_by_sex_education,
+                "hlth_ehis_pe2i":
+                    healthenhancing_nonworkrelated_by_sex_income,
+                "hlth_ehis_pe2u":
+                    healthenhancing_nonworkrelated_by_sex_urbanisation,
+                "hlth_ehis_pe9b":
+                    healthenhancing_by_sex_nativity,
+                "hlth_ehis_pe9c":
+                    healthenhancing_by_sex_citizenship,
+                "hlth_ehis_pe9d":
+                    healthenhancing_by_sex_activitylimitation,
+                "hlth_ehis_pe2m":
+                    healthenhancing_nonworkrelated_by_sex_bmi,
+                "hlth_ehis_de9":
+                    dailypractice_by_sex_education
+            }
+            df = function_dict[file_name](df)
+            df['SV'] = df['SV'].str.replace('_Total', '')
+            final_df = pd.concat([final_df, df])
+            sv_list += df["SV"].to_list()
+
+        final_df = final_df.sort_values(by=['time', 'geo', 'SV', 'observation'])
+        final_df = final_df.drop_duplicates(subset=['time','geo','SV'],\
+            keep='first')
+        final_df['observation'] = final_df['observation'].astype(str)\
+            .str.strip()
+        # derived_df generated to get the year/SV/location sets
+        # where 'u' exist
+        derived_df = final_df[final_df['observation'].astype(str).str.contains(
+            'u')]
+        u_rows = list(derived_df['SV'] + derived_df['geo'])
+        final_df['info'] = final_df['SV'] + final_df['geo']
+        # Adding Measurement Method based on a condition
+        final_df['Measurement_Method'] = np.where(
+            final_df['info'].isin(u_rows),
+            'EurostatRegionalStatistics_LowReliability',
+            'EurostatRegionalStatistics')
```

Comment:
Expand comment more to provide informtion on LowReliabiility and how Time/Geo series will be impacted. Please make it generic (e.g., using dictionary - for now it can have only 'u' value) to handle other scenarios for future use case.